tests-behat/revealEye.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Password with revealEye
+
+  Scenario:
+    Given I am on "form-control/input2.php"
+    When I fill in "password_re_norm" with "123"
+    Then "password_re_norm" should contain text "123"
```

Comment:
I think the text will be always contained, you need to check if the input `type` attribute is changed.

Reply:
I know, this is just trial and error...
I've got to see how to do a class check for the icon.

Reply:
done